Calums_README.md

```diff
@@ -0,0 +1,16 @@
+## APPROACH
+
+In this challenge, I have followed a similar approach to that which we undertook in the weekly challenge of building a bookmark manager app.
+
+
+
+
+### To set up the database
+
+ Connect to `psql` and create the `peeps` database:
```

Comment:
I noticed the database name is the same as the table name - should this be 'chitter'?